hta_page.qmd

```diff
@@ -0,0 +1,22 @@
+---
+title: "HTA-R workstream"
+---
+
+**Lead**: Gregory Chen (MSD)
+
+**Objective**: 
+This workstream in SWE WG unites voluntary statistical software engineers that are experienced in HTA (Health Technology Assessment) or highly-interested in this area. 
+The team aims to develop open-source R tools of good quality in the right formats (R pkg, apps, user guides) that are useful to both manufacturers and payers, to support crucial analytic topics in HTA dossier submission across various countries, particularly the topics with unmet needs in R implementation and/or related to [EUnetHTA](https://www.eunethta.eu/).
+
+The workstream also works within a broader group (link tbd) of business and methodological SMEs in HTA and Market Access space of our pharma industry, to ensure this open-source HTA pipeline of R tools is relevant over time and the tools can be adopted easily by business and payers.   
```

Comment:
I think there is no need to refer to a "link tbd" if that doesn't exist – can always be added later?

Reply:
Yeah I would agree. Also from strategy perspective, it might be easier to think about collaboration SWE WG - HTA WGs rather than this workstream being wrapped into yet another group?

Reply:
link tbd is deleted. We can discuss more about the phrasing about the connection between this wg with the broader group. The broader group has a meeting at March 8.